src/current/_includes/v23.2/ui/insights.md

```diff
@@ -167,17 +186,23 @@ The Workload Insights tab surfaces the following type of insights:
 The transaction or statement execution failed.
 
 {% if page.cloud != true -%}
-The following screenshot shows a failed statement execution:
+The following screenshot shows a failed transaction execution:
+
+<img src="{{ 'images/v23.2/transaction_executions_failed.png' | relative_url }}" alt="Failed transaction execution" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v23.2/statement_executions_failed.png' | relative_url }}" alt="Failed statement execution" style="border:1px solid #eee;max-width:100%" />
+The following screenshot shows the default details of the preceding failed transaction execution.
 
-The following screenshot shows the execution details of the preceding failed statement execution.
+<img src="{{ 'images/v23.2/transaction_execution_details_failed.png' | relative_url }}" alt="Failed transaction execution details" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v23.2/statement_execution_details_failed.png' | relative_url }}" alt="Failed statement execution details" style="border:1px solid #eee;max-width:100%" />
+The following screenshot shows the conditional details of the preceding failed transaction execution.
+
+<img src="{{ 'images/v23.2/transaction_execution_details_failed_conditional.png' | relative_url }}" alt="Failed transaction execution details conditional" style="border:1px solid #eee;max-width:100%" />
 
 {% endif -%}
 
-The **Insights** column shows the name of the insight, in this case **Failed Execution**. The **Details** column provides the **Error Code**, such as `40001`. CockroachDB uses [PostgreSQL Error Codes](https://www.postgresql.org/docs/current/errcodes-appendix.html). `40001` indicates a `serialization_failure`.
+The **Insights** column shows the name of the insight, in this case **Failed Execution**. The **Details** column provides the **Error Code** and **Error Message**. CockroachDB uses [PostgreSQL Error Codes](https://www.postgresql.org/docs/current/errcodes-appendix.html).
+
+To capture more information in the event of a failed transaction execution due to a serialization conflict, set the `sql.contention.record_serialization_conflicts.enabled` [cluster setting]({{ link_prefix }}cluster-settings.html) to `true`. With this setting enabled, when the **Error Code** is `40001` and the **Error Message** specifically has [`RETRY_SERIALIZABLE`]({{ link_prefix }}transaction-retry-error-reference.html#retry_serializable)` - failed preemptive refresh due to conflicting locks`, a conditional **Failed Execution** section is displayed with **Conflicting Transaction** and **Conflicting Location** information.
```

Comment:
We should call out (perhaps as another conditional?) that conflicting transaction id is not ALWAYS captured and list out the conditions when it's not (value committed, other?).

@aadityasondhi's doc:
https://docs.google.com/document/d/1LS7AmXjzCdTNe2B0WwRgPFg_7OU3h8NyEDaWYmnLkoU/edit#heading=h.hl683772fu4x

Reply:
Refer to this explanation: https://github.com/cockroachdb/docs/pull/17992#discussion_r1378007752

My doc is a little outdated since we renamed some of these concepts at the lower transactions level.

Reply:
see [above comment](https://github.com/cockroachdb/docs/pull/17992#discussion_r1382223611)